apps/product/views.py

```
@@ -1,7 +1,201 @@
-
+from drf_spectacular.utils import extend_schema, OpenApiTypes, OpenApiParameter, OpenApiExample
 from django.shortcuts import get_object_or_404
 from rest_framework import status, permissions
 from rest_framework.response import Response
+from rest_framework.request import Request
 from rest_framework.views import APIView
-from drf_yasg.utils import swagger_auto_schema
-from drf_yasg import openapi
+from apps.product.serializer import ClasterSerializer, WarehouseSerializer
+from rest_framework import status
```

Ruff

Redefinition of unused `status` from line 3

Remove definition: `status`

(F811)